netbench-orchestrator/src/orchestrator/cli.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::orchestrator::{OrchError, OrchResult};
+use serde::Deserialize;
+use std::{
+    fs::File,
+    path::{Path, PathBuf},
+};
+
+#[derive(Clone, Debug)]
+pub struct OrchestratorConfig {
+    // netbench
+    netbench_scenario_filename: String,
+    netbench_scenario_filepath: PathBuf,
+
+    // cdk
+    pub cdk_config: CdkConfig,
+}
+
+// Used for parsing the scenario file generated by the s2n-netbench project
+#[derive(Clone, Debug, Default, Deserialize)]
+pub struct CdkConfig {
+    #[serde(rename(deserialize = "NetbenchInfraPrimaryProd"))]
+    resources: CdkResources,
+}
+
+#[derive(Clone, Debug, Default, Deserialize)]
+#[serde(rename_all(deserialize = "camelCase"))]
+struct CdkResources {
```

Comment:
Could we document what these represent. E.g. "The netbench_cloudfront_distribution holds all netbench reports that are generated by the driver processes" or seomthing like that.